### PR TITLE
Jobs

### LINKED ISSUE #16: Create Jobs model and DB table
- [x] Create branch
- [x] Write test
- [x] Create migration
- [x] Create model
- [x] Push code

### PR DESCRIPTION
**Pull request by bplantico**
## Changes proposed in this pull request:
Adds Jobs model and table. Jobs has a name and timestamps.

## Card(s) closed in this pull request:
close #16 

## What did you struggle on to complete?
Didn't struggle with anything on this, but almost forgot to create the appropriate associations so had to go back and add a garden_id reference to the jobs table.

### Model Test Coverage: 100%
- [x] Tested my new feature(s) as well as any feasible edge cases (if possible)
- [x] Checked coverage/index.html - did not add any new code that's not covered by testing (if possible)
- [x] Ran the test suite - all tests are passing
- [x] Merged in the latest master to my branch with git pull origin master & resolved merge conflicts
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: spec/models/garden_spec.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe Garden, type: :model do
 	describe "relationships" do
-		it { should have_many(:env_measurements) }
+		it { should have_many :env_measurements }
```
Comment (CosmicSpagetti) [inline spec/models/garden_spec.rb]: Oh nice that parenthesis really bothered me too!
